### PR TITLE
Feature: Badge Demo

### LINKED ISSUE #27: Add colored badges to badge demo
Add colored badges to badge demo when #22 is accepted.

### PR DESCRIPTION
**Pull request by CalebBassham**
Added a demo for a simple badge. Will go back and add colored badges when #22 is accepted.

Closes #27

### CODE REVIEW

File: demo/page/component/Badge.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template lang="html">
+  <div>
+    <Demo title="Simple badge">
+      <h4>Header <p-badge>5</p-badge></h4>
```
Comment (TotomInc) [inline demo/page/component/Badge.vue]: Can you add multiple examples of badges colors please? Like it was was done on [getpapercss.com](https://www.getpapercss.com/#badges)

Comment (CalebBassham) [reply demo/page/component/Badge.vue]: #27

**Comment by DarkPaladinUHC** [comment]
Closes #27

**Comment by CalebBassham** [comment]
image

**Comment by CalebBassham** [comment]
image

**Review by TotomInc — APPROVED** [review]

**Pull request merged**